packages/fe-mockserver-core/src/request/odataRequest.ts

```diff
@@ -39,6 +39,34 @@ export type QueryPath = {
     keys: KeyDefinitions;
 };
 
+function addPathToExpandParameters(
+    path: string,
+    expandParameter: Record<string, ExpandDefinition>,
+    lambdaVariable?: string,
+    skipLast?: boolean,
+    removeFromResult?: boolean
```

Comment:
I think either this should be `removeFromResult = true` or https://github.com/SAP/open-ux-odata/pull/763/files#diff-375923a6620bad4a6bf788b20f73ebd5fa42227e0109b150c2614ea1450e3be4R581 should set `removeFromResult` to `true` in order to keep the previous behaviour with lambda expressions